concordia/views.py

```diff
@@ -1118,6 +1128,73 @@ def reserve_asset_transcription(request, *, asset_pk):
     return HttpResponse(status=204)
 
 
+@never_cache
+@atomic
+def redirect_to_next_reviewable_asset(request, *, campaign_slug):
+    campaign = get_object_or_404(Campaign.objects.published(), slug=campaign_slug)
+    project_slug = request.GET.get("project", "")
+    item_id = request.GET.get("item", "")
+    asset_id = request.GET.get("asset", 0)
+
+    if not request.user.is_authenticated:
+        user = get_anonymous_user()
+    else:
+        user = request.user
+
+    potential_assets = Asset.objects.select_for_update(skip_locked=True, of=("self",))
+    potential_assets = potential_assets.filter(
+        item__project__campaign=campaign,
+        item__project__published=True,
+        item__published=True,
+        published=True,
+    )
+    potential_assets = potential_assets.filter(
+        transcription_status=TranscriptionStatus.SUBMITTED,
+        assettranscriptionreservation=None,
```

Comment:
The `assettranscriptionreservation` shouldn't be relevant for review since that's only used when editing.